src/olympia/devhub/templates/devhub/nav.html

```diff
@@ -55,7 +55,7 @@ <h1 class="site-title prominent">
           {{ _('Add-ons Blog') }}</a></li>
         <li><a href="https://discourse.mozilla-community.org/c/add-ons">
           {{ _('Add-ons Forum') }}</a></li>
-        <li><a href="https://developer.mozilla.org/en-US/Add-ons">
+        <li><a href="https://extensionworkshop.com/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=devhub-menu-link#connect">
```

Comment:
just going to wait for answer on https://github.com/mozilla/addons-server/issues/10910#issuecomment-475288915